ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,72 @@
+Issue Title
+===========
+
+Your issue title is above. Please include the path to the
+topic with the issue. For example, if your issue is that
+our [list of samples](https://stage.docs.microsoft.com/en-us/dotnet/articles/samples-and-tutorials/index)
+is missing a sample, you'd put [samples-and-tutorials/index.md]
+in the issue title.
+
+If your issue is that a topic is missing, please preface
+the issue title with [NewTopic]. That helps us find requests
```

Comment:
I'm not sure we need to have customers set the title to path/doc.md, and even if we ask them I'm guessing few will follow this. But I think it's critical that issues are stored that way for easy searching and to prevent duplicates. They is frequently an issue open and you want to add a task list
- [ ] Fix xyz
- [ ]  add section on ABC

It doesn't hurt to ask folks to use this format. Not organizing topic issues this way makes it difficult to manage issues.